src/actions/showNotification.ts

```diff
@@ -0,0 +1,39 @@
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+
+/**
+ * Show Notification Action. Displays a local notification.
+ *
+ * ```js
+ * showNotification({
+ *  title: 'Hello, World!',
+ *  body: 'This is a test notification',
+ *  sound: true,
+ * });
+ * ```
+ */
+
+const showNotification = (
+  options: {
+    /* Title for the notification */
```

Comment:
Could the start of this comment be changed to `/**` please? Also, could you add similar comments for `body` and `sound`, just explaining in a couple of words what the do. 🙂

Reply:
Comment format updated & comments added for body and sound!